Packages.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+	<ItemGroup>
+		<PackageReference Update="Newtonsoft.Json" Version="[6.0.6]" />
```

Comment:
Just starting with this one for now, I want a `-preview` build of stuff with 7.0.1 (follow-up PR)

Reply:
Oh, it's all or nothing eh